workspaces/optic-engine-wasm/src/lib.rs

```diff
@@ -11,7 +11,7 @@ use optic_engine::{
 };
 use std::collections::HashMap;
 use uuid::Uuid;
-use wasm_bindgen::prelude::*;
+use wasm_bindgen::{__rt::WasmRefCell, prelude::*};
```

Comment:
Not sure how this got here, might be an auto import that needs to be cleaned up 🤔